### DISCUSSION TITLE
I've made a few small bugfixes inside spaghetticode.js which I would love to share with you

### DISCUSSION THREAD

**Discussion by Christian-Wania** [SupaGruen/SD-CheatSheet#1] [Show and tell]
Since I can't push them directly I'll try it this way ;-)

The Changes are: 

- Escape special characters in the HTML output to prevent <lora, <lyco, and other "tags" from disappearing inside the prompts on the metadata page.
- 
- "Copy Prompt" copies the original (unparsed) prompt information of the image. This prevents the destruction of additional data which might have been added from Stable Diffusion extensions (e.g. sd-dynamic-prompts).
If additional data is available, it will be displayed incorrectly on the metadata page, but it can be transferred cleanly with this change.

The attached image contains additional data from sd-dynamic-prompts, and the txt file contains my changes to your code.

[spaghetticode.js.txt](https://github.com/SupaGruen/StableDiffusion-CheatSheet/files/11523467/spaghetticode.js.txt)
![00354-3102309693](https://github.com/SupaGruen/StableDiffusion-CheatSheet/assets/6764186/6dcfb7c2-d828-4aeb-9e75-2ab1de1d19c3)

**Comment by SupaGruen** [comment]
Hi @Christian-Wania, thanks for making me aware! Unencoded LoRA tags were not on my radar. I checked your file, but couldn't see any changes (and it was also an older version of the JS), so I wrote some new code and everything should be covered now.

- [x]  encoded special characters in metadata output
- [x]  support for the extra "Template" metadata (we'll have to see if that covers all the other cases that might come)
- [x] "copy prompt" with original metadata

I'm currently working on the next release (1.1.0) but it's not ready yet.
For now, you can test the changes [here](https://supagruen.github.io/StableDiffusion-CheatSheet/) and of course download the updated "spaghetticode.js", I haven't checked if it breaks the old release though, but hopefully it doesn't!

**Reply by Christian-Wania** [reply]
Damn, it seems like I uploaded the wrong file, sorry 🤦‍♂️
Your recent update looks great, and for me it's working 👍
Thanks 😊
